ush/templates/FV3LAM_wflow.xml

```diff
@@ -2212,6 +2212,86 @@ MODULES_RUN_TASK_FP script.
 {%- endif %}
 
 {%- if do_dacycle %}
+{%- if do_sddacycle %}
```

Comment:
There is another option "DO_PM_DA". Please merge "do_sddacycle" and "DO_PM_DA" option as one option.

Reply:
Thanks. I will look into this tomorrow.

Reply:
@hu5970 
Please review the changes according to the discussion yesterday. 
Thanks,
Hongli